README.md

```diff
@@ -22,6 +22,9 @@ that any file that passed these schemas should be accepted by Ansible.
 - Inline actions are not allowed, as schema cannot validate them
 - Non builtin modules must be called using `action:` blocks
 - Module arguments are not yet verified but we plan to implement it
+- Out schemas are strict about use of jinja2 templating and require `{{` on
+  arguments declared as **explicit**, which forbid use of `{{` on those marked
+  as **implicit**. See section below for details.
```

Comment:
not forbid, but discourage, as they lead to double interpolation which is not what people normally intend .. except for a couple that specifically intend it

Reply:
explicit do not 'require' {{ }} , as they can take straight values, but do require the moustaches to process variables and expressions:
```
run_once: true
```
vs

```
run_once: {{varthatisboolean}}
```

Reply:
Keep in mind that a schema does not have warning levels, it can either be accepted or not, nothing in-between.

Also, `true` is a boolean not a string. So for strings, we would request strings with mustaches for fields marked as explicit.

Reply:
I would also define 'static' as no templating (see register)